packages/wonder-blocks-cell/src/util/types.js

```diff
@@ -132,4 +132,10 @@ export type CellProps = {|
      * by default if react-router is present.
      */
     href?: string,
+
+    /**
+     * A target destination window for a link to open in. Should only be used
+     * when `href` is specified.
+     */
+    target?: "_blank",
```

Comment:
Should this be a variation of the `href` prop instead, since it's only relevant when that's set? For example:

```js
    href?: string | {|href: string, target: "_blank"|},
```